controller/tests/controller_server/csi_controller_server_test.py

```diff
@@ -130,17 +131,15 @@ def test_create_snapshot_succeeds(self, storage_agent):
         self.servicer.CreateSnapshot(self.request, self.context)
 
         self.assertEqual(self.context.code, grpc.StatusCode.OK)
-        self.mediator.get_snapshot.assert_called_once_with(snapshot_name, pool_id=None)
-        self.mediator.create_snapshot.assert_called_once_with(snapshot_name, snapshot_volume_name, None)
+        self.mediator.get_snapshot.assert_called_once_with(snapshot_name, pool_id="pool1")

```

Comment:
consider adding `"pool1"` to `test_settings.py`

Reply:
done